README.md

```diff
@@ -58,6 +58,14 @@ Binaries are generated from `main` branch every night for `Linux` and `Windows`.
 
 Please be aware: nightly builds might have critical bugs, it's not recommended for use in production and no support provided.
 
+## CI Dashboard
```

Comment:
My only concern with calling this the "CI Dashboard" is that it only reflects a couple of more than 40 CI actions on the branch head or each pull request. Should we call it the "K8s CI Dashboard" since it is only the Kubernetes->CRI aspect of containerd testing?

Reply:
The readme already contains a badge for CI and Nightly builds.  Thats why skipped adding it here.  

But for this I think k8s-CRI also makes sense.

Reply:
@estesp updated the wording , can you take another look.